utils/string_utils.js

```diff
@@ -6,9 +6,20 @@ const wordsStandardize = (word) => {
 	return 'null';
 };
 
+const wordToBoolean = word => {
+	if (word === '') return true;
+	if (word) {
+		if (word.toLowerCase() === 'true' || word === '1') return true;
+		else if (word.toLowerCase() === 'false' || word === '0') return false;
+		else return Boolean(word);
+	}
+	return false;
+};
```

Comment:
To be honest, I think we should keep query params to two options: `?yesterday` and `yesterday=true`. There isn't really a need for `0` or `1`